src/Components/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+import types from '../types/types';
+
+// initial state
+const initialState = {
```

Comment:
- [ ] According to the project requirement, your file structure should be something like this 👇🏾 
![image](https://user-images.githubusercontent.com/84156714/214308419-65e986c8-c803-4867-bbcc-3ea259a7ca93.png)

Please fix this.